pkgs/jni/CHANGELOG.md

```diff
@@ -23,11 +23,11 @@
 - Fixed a bug where reading non-null terminated strings would overflow.
 
 ## 0.7.1
-- Removed macOS Flutter plugin until package:jni supports it ([#41](https://github.com/dart-lang/jnigen/issues/41)).
+- Removed macOS Flutter plugin until package:jni supports it ([#780](https://github.com/dart-lang/native/issues/780)).
```

Comment:
The old links are forwarded, so it's not needed, but since you've already migrated it, we can keep it.

Reply:
That's how I changed them even!